guides/source/configuring.md

```diff
@@ -396,637 +560,930 @@ in controllers and views. This defaults to `false`.
 
-* `config.active_record.query_log_tags` define an `Array` specifying the key/value tags to be inserted in an SQL comment.
-  Defaults to `[ :application ]`, a predefined tag returning the application name.
+Allows specifying the job that will be used to destroy the associated records in background. It defaults to `ActiveRecord::DestroyAssociationAsyncJob`.
 
-* `config.active_record.cache_query_log_tags` specifies whether or not to enable caching of query log tags.
-  For applications that have a large number of queries, caching query log tags can provide a performance benefit when
-  the context does not change during the lifetime of the request or job execution.
-  Defaults to `false`.
+#### `config.active_record.queues.destroy`
 
-* `config.active_record.schema_cache_ignored_tables` define the list of table that should be ignored when generating
-  the schema cache. It accepts an `Array` of strings, representing the table names, or regular expressions.
+Allows specifying the Active Job queue to use for destroy jobs. When this option is `nil`, purge jobs are sent to the default Active Job queue (see `config.active_job.default_queue_name`). It defaults to `nil`.
+
+#### `config.active_record.enumerate_columns_in_select_statements`
+
+When true, will always include column names in `SELECT` statements, and avoid wildcard `SELECT * FROM ...` queries. This avoids prepared statement cache errors when adding columns to a PostgreSQL database for example. Defaults to `false`.
+
+#### `config.active_record.destroy_all_in_batches`
+
+Ensures ActiveRecord::Relation#destroy_all to perform the record's deletion in
+batches. ActiveRecord::Relation#destroy_all won't longer return the collection
+of the deleted records after enabling this option.
+
+#### `config.active_record.verify_foreign_keys_for_fixtures`
+
+Ensures all foreign key constraints are valid after fixtures are loaded in tests. Supported by PostgreSQL and SQLite only. Defaults to `false`.
+
+#### `config.active_record.query_log_tags_enabled`
+
+Specifies whether or not to enable adapter-level query comments. Defaults to
+`false`.
+
+#### `config.active_record.query_log_tags`
+
+Define an `Array` specifying the key/value tags to be inserted in an SQL
+comment. Defaults to `[ :application ]`, a predefined tag returning the
+application name.
+
+#### `config.active_record.cache_query_log_tags`
+
+Specifies whether or not to enable caching of query log tags. For applications
+that have a large number of queries, caching query log tags can provide a
+performance benefit when the context does not change during the lifetime of the
+request or job execution. Defaults to `false`.
+
+#### `config.active_record.schema_cache_ignored_tables`
+
+Define the list of table that should be ignored when generating the schema
+cache. It accepts an `Array` of strings, representing the table names, or
+regular expressions.
 
 The MySQL adapter adds one additional configuration option:
```

Comment:
Thanks for spotting this! It also applies to the Schema Dumper.
Adding headings could make it look like this:
<img width="691" alt="image" src="https://user-images.githubusercontent.com/28561/131890185-8f6f2da5-0545-4cd0-9825-5eee675d01ad.png">

Mentioning the database could make it look like this (I slightly prefer this):
<img width="695" alt="image" src="https://user-images.githubusercontent.com/28561/131891754-bb7e56b1-8972-414d-b0db-563e6808559a.png">

Reply:
I think they both look fine, so I don't have a strong preference.  If we go with the first option though, it might be good to move those sections to just above [Configuring a Database](https://guides.rubyonrails.org/v6.1/configuring.html#configuring-a-database), to preserve the flow of "Configuring Active ..." sections.

Reply:
I went with the second option, to keep things simple.